Tests/SonarScanner.MSBuild.PostProcessor.Test/PostProcessorTests.cs

```diff
@@ -188,7 +190,7 @@ public void PostProc_WhenNoSettingInFileAndCommandLineArg_Fail()
             context.TfsProcessor.ValueToReturn = false;
 
             // Act
-            var success = Execute(context, true, args: "/d:sonar.login=foo");
+            var success = Execute(context, true, args: "/d:sonar.token=foo");
```

Comment:
This should work for both `sonar.token` and `sonar.login`, right?
I think it would be useful to have a data-driven testcase that passes both of them (one at a time).

Reply:
I think we have lots of tests that check the properties and this test is concerned about the lack of settings and not about authentication credentials.